Defs/ThingDefs_Buildings/Buildings_Turrets.xml

```diff
@@ -377,10 +377,15 @@
       <Mass>1000</Mass>
       <Bulk>1000</Bulk>
     </statBases>
-    <costList>
-      <Steel>500</Steel>
-      <ComponentIndustrial>5</ComponentIndustrial>
-    </costList>
+    <costListForDifficulty>
+      <difficultyVar>classicMortars</difficultyVar>
+      <invert>true</invert>
+      <costList>
+        <ComponentIndustrial>5</ComponentIndustrial>
+        <ReinforcedBarrel>1</ReinforcedBarrel>
+      </costList>
+      <Steel>350</Steel>
```

Comment:
This throws a start-up error. I'm still looking into the structure of how exactly the structure of this tag works, so I'm not sure what the correct format is.